src/Datadog.Trace/ClrProfiler/DynamicMethodBuilder.cs

```diff
@@ -14,6 +15,8 @@ namespace Datadog.Trace.ClrProfiler
     /// </summary>
     /// <typeparam name="TDelegate">The type of delegate</typeparam>
     [Obsolete("This type will be removed in a future version of this library.")]
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
     public static class DynamicMethodBuilder<TDelegate>
```

Comment:
I think we can make this one `internal`.
```suggestion
    internal static class DynamicMethodBuilder<TDelegate>
```

Reply:
It's never used, and as it's now moved assembly, should we just remove it?